### PR TITLE
[ng] API homepage: show import statement for each library

### PR DESCRIPTION
**Pull request by chalin**
Contributes to #787 -- see https://github.com/dart-lang/site-webdev/issues/787#issuecomment-317121588 for screenshots.

Staged at https://angulardart-org-dev.firebaseapp.com/angular/api (you might have to clear the cache and/or open in an incognito window).

### CODE REVIEW

File: src/resources/js/directives/api-list.js [hunk 1]
```diff
@@ -46,7 +46,15 @@ angularIO.directive('apiList', function () {
       '      ' +
       '<article class="l-content-small docs-content">' +
       '  <div ng-repeat="section in $ctrl.groupedSections" ng-if="$ctrl.filterSections(section)" ng-cloak="ng-cloak">' +
-      '    <h2>{{ section.title }}</h2>' +
+      '    <a ng-href="/angular/api/{{section.title}}/{{section.title}}-library" target="_blank" rel="noopener">' +
+      '      <h2>' +
+      '        {{section.title}}' +
+      '          <span style="vertical-align:middle;">&#8227</span>' +
```
Comment (kwalrath) [inline src/resources/js/directives/api-list.js]: I love it except for the icon, which looks like it should open something. Other possibilities:

* nothing (except maybe more space)
* a colon (:)
* a non-triangular arrow such as →

Comment (kwalrath) [inline src/resources/js/directives/api-list.js]: The mobile version shouldn't have an icon. And I think we really don't need one in the desktop version either. If that looks to weird, we could put it below the library name, as in the mobile version.

Comment (chalin) [inline src/resources/js/directives/api-list.js]: Ok. let's try dropping the triangle in favor of a little more space.

**Comment by chalin** [comment]
Changes made:
- There is now only space between the library name and its import statement.
- Only the library name is a link to its API entry
  The import statement, no longer being a link, is more easily selectable.

Staged server refreshed: https://angulardart-org-dev.firebaseapp.com/angular/apiStaged at https://angulardart-org-dev.firebaseapp.com/angular/api

Here's the updated look:
> screen shot 2017-07-22 at 7 18 38 am

> screen shot 2017-07-22 at 7 19 24 am

**Pull request merged**